charts/renku/values.yaml

```diff
@@ -540,18 +547,26 @@ notebooks:
         #        secretKeyRef:
         #          name: renku-jupyterhub-postgres
         #          key: jupyterhub-postgres-password
-    
+
     proxy:
       ## Must be set explicitly, generate using `openssl rand -hex 32`
-      secretToken: 
+      secretToken:
 
-    ## Simple default setup for gitlab oauth
-    # auth:
+    ## Configuration for GitLab acting as OAuth2 provider for Jupyterhub.
+    auth:
```

Comment:
```suggestion
    ## Configuration for GitLab acting as OAuth2 provider for Jupyterhub.
    ## If deploying against an existing GitLab, the OAuth2 application has to be created 
    ## manually and the corresponding values added to the fields below.
    auth:
```

Reply:
These values always have to be set explicitly. We can currently not auto-create them. For this we will first have to tackle https://github.com/SwissDataScienceCenter/renku/issues/1490.

I'm not claiming that this PR improves the documentation aspect in this values file by much, I just wanted to quickly add references to the `cookieSecret` and the `cryptoKey` which both need to be set by the user but were not mentioned here at all.

Reply:
it's been a long time since I've deployed renku from scratch with gitlab, but my impression was that we create those apps in gitlab automatically? Is that not the case anymore?

Reply:
Yes, that is the case. But the above block configures Jupyterhub to *use* GitLab as OAuth2 provider. I just adapted the wording a bit to make this clear.

Reply:
I just figured the user will wonder where to find these values but ok we can leave it unspecified.